README.md

```diff
@@ -21,34 +21,41 @@ You can find the wiki [here](https://github.com/fleekxyz/non-fungible-apps/wiki)
 ### 📁 Project Structure
 
 Inside the root folder you are going to find:
+- [/contracts](/contracts): all the developed contracts
+- [/subgraph](/contracts): The Graph project related code
+- [/ui](/ui): a web application to interact with deployed contracts
 
+Inside contracts you are going to find:
 - [/contracts](/contracts): all the developed contracts
 - [/deploy](/deploy): scripts used for deployment on different networks
 - [/deployments](/deployments): resultant ABI and info for deployments (each network will have a nested folder)
 - [/lib](/lib): external modules used by Foundry
 - [/scripts](/scripts): any utility scripts used for interacting with deployed contracts
 - [/test](/test): tests suits to validate contracts
-- [/ui](/ui): a web application to interact with deployed contracts
 
 And after running it locally some folders may be generated:
 
-- `/artifacts`: ABIs and build info generated by Hardhat
-- `/cache`: cache info used by Hardhat
-- `/forge-cache`: cache info used by Foundry
-- `/node_modules`: all dependencies for the Node.js environment
-- `/out`: resultant ABIs for all contracts that has interactions
+- `/contracts/artifacts`: ABIs and build info generated by Hardhat

```

Comment:
Maybe we could have `README`s inside of each folder with the details of each one inside of it.